bento_sdk/bento_sdk.info.yml

```
@@ -0,0 +1,7 @@
+name: 'Bento SDK'
+type: module
+description: 'Official Bento API integration for Drupal 10'
+core_version_requirement: ^10.2
+package: 'Bento'
+dependencies:
+  - drupal:system
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)